prototyping/wlutz/testing-previously-failing-images.ipynb

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
Is the idea that they should continue to fail?
Or to prove that with some new whiz-bang algorithm, it no longer fails?

Reply:
There are two subsets of cases. One where it really should have succeeded, but didn't. And some where it should have failed fast and instead it ran in with misplaced confidence.

---

Here is an example case where PyMedPhys should have handled the artefacts, but previously it didn't. Some algorithm adjustments now allows most of these to work:

![image](https://user-images.githubusercontent.com/6559099/103146706-48b41400-47a1-11eb-84da-c3615ac022b1.png)

That image above is what is being tested here:

https://github.com/pymedphys/pymedphys/blob/4c5cfdb17ad2a6eaaaee9e7bba4f029183a4eeeb/lib/pymedphys/tests/experimental/wlutz/test_previous_failure_modes.py#L76-L83

---

The cases where it used to run in, over-confident, when it shouldn't have are in the following file:

https://github.com/pymedphys/pymedphys/blob/4c5cfdb17ad2a6eaaaee9e7bba4f029183a4eeeb/lib/pymedphys/tests/experimental/wlutz/test_should_fail.py#L16-L18

Here is a collection of weird images (produced by the iView during the first and last frames of a movie usually) where I would rather PyMedPhys produce no result at all than attempt it with a misplaced confidence. Here are some examples of those images:

![image](https://user-images.githubusercontent.com/6559099/103146621-62a12700-47a0-11eb-989f-0eb536c4c636.png)

![image](https://user-images.githubusercontent.com/6559099/103146626-6a60cb80-47a0-11eb-90c1-085678787e3e.png)